addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -783,6 +783,7 @@ $o-bg-shapes: ('web_editor': (
     'Zigs/03': ('position': top, 'size': 100% auto, 'colors': (1), 'repeat-y': true),
     'Zigs/04': ('position': bottom, 'size': 100% auto, 'colors': (1), 'repeat-y': false),
     'Zigs/05': ('position': bottom, 'size': 100% auto, 'colors': (3), 'repeat-y': false),
+    'Zigs/06': ('position': bottom, 'size': 30px 100%, 'colors': (4, 5), 'repeat-x': true),
```

Comment:
Mmh, to be consistent we should define `'repeat-x': false` everywhere else... I'll do the opposite and remove all the `'repeat-y': false`.